frontend/src/app/model/player.ts

```diff
@@ -3,14 +3,15 @@ import {Avatar} from '../util/avatar';
 import {Color} from '../util/color';
 
 export class Player {
-  private system = false;
+  private _system = false;
```

Comment:
no one uses _ to mark private members in 2019

Reply:
It's an official TypeScript approach for getters and setters for properties, see the [doc](https://www.typescriptlang.org/docs/handbook/classes.html). But maybe we don't need a getter for the system, need to check.

Reply:
In that doc they used _ in one of example, in others private props was named same as others